Samples/PowerShell/SAM.AppInsightAutomation/SetupAppInsightApplication.ps1

```diff
@@ -0,0 +1,34 @@
+﻿Import-Module SwisPowerShell
+
+#orion machine settings
+$hostname = "hostname"
+$username = "Admin"
+$plainTextPassword = "a"
+$password = $plainTextPassword | ConvertTo-SecureString -AsPlainText -Force
+
+$cred = New-Object -typename System.Management.Automation.PSCredential -argumentlist $username, $password
+$swis = Connect-Swis -host $hostname -cred $cred
+
+#target node settings
+$nodeName = 'nodeName'
+$nodeIp = 'nodeIp'
+$nodeId = 1
+$credentialSetId = 1
+
+#$template = "AppInsight for Exchange"
+$template = "AppInsight for IIS"
```

Comment:
Do we need a comment to explain the available template options here?

Reply:
comment removed